docs/api/v1/components/responses/repo_link.yaml

```diff
@@ -0,0 +1,4 @@
+type: object
```

Comment:
duplicate of [`docs/api/v1/components/schema/repo_link.yaml`](https://github.com/forgefedv2/interface/blob/2366e3efb60ae8b8ef860f263cf267b1ec35354d/docs/api/v1/components/schema/repo_link.yaml)

Reply:
Noted, will be removing the duplicate object.